src/views/refund/refund.js

```diff
@@ -24,6 +24,18 @@ const styles = theme => ({
   },
 });
 
+const uploadRefundFileText = (refundFile, txHash) => {
+  const invalidRefundFile = Object.keys(refundFile).length === 0;
+  const invalidTxHash = txHash === '';
+  if (invalidRefundFile && !invalidTxHash) {
+    return 'Upload refund file';
+  } else if (!invalidRefundFile && invalidTxHash) {
+    return 'Input transaction hash';
+  } else {
+    return 'Upload refund file';
```

Comment:
The more I think about it the weirder this if statements get :thinking: 

Wouldn't a check for the refund file:

```
if (Object.keys(refundFile).length === 0) {
  return 'Upload refund file';
}
```

and afterwards a check for the transaction hash:

```
if (txHash === '') {
  return 'Input transaction hash';
}
```

be a lot more elegant and also get rid of the variables `invalidRefundFile` and `invalidTxHash`?